src/connector/src/source/filesystem/opendal_source/mod.rs

```diff
@@ -38,10 +38,15 @@ pub const POSIX_FS_CONNECTOR: &str = "posix_fs";
 pub struct GcsProperties {
     #[serde(rename = "gcs.bucket_name")]
     pub bucket_name: String,
+
+    /// The base64 encoded credential key. If not set, ADC will be used.
```

Comment:
Although both methods can be used for permission verification, I recommend that this be consistent with the object store. That is keeping using `builder.credential(&cred);` instead of `builder.credential_path(&cred_path);`

Reply:
object store does not have a field for credential, it use `ADC`  by default. so let's use `credential `